modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -393,7 +393,7 @@ EditorSceneImporterAssimp::_generate_scene(const String &p_path, aiScene *scene,
 				ERR_FAIL_COND_V_MSG(parent_node == NULL, state.root,
 						"Parent node invalid even though lookup successful, out of ram?")
 
-				if (parent_node && spatial != state.root) {
+				if (spatial != state.root) {
 					parent_node->add_child(spatial);
 					spatial->set_owner(state.root);
 				} else if (spatial == state.root) {
```

Comment:
Next run of your static analyzer it will complain about this, so best fix it now too (the `else if` should simply be `else`, and the current `else` branch is dead code).